backend/requirements-dev.txt

```diff
@@ -0,0 +1,231 @@
+aiohttp==3.9.5
```

Comment:
The usual convention is `requirements-test.txt` - test only packages (mypy, types, ruff, etc) and `requirements.txt` (bare minimum what is needed for prod)

Reply:
I've renamed `requirements-dev.txt` to `requirements-test.txt`. Currently, `requirements.txt` only contains the necessary dependencies. I have verified this by installing the deps manually on a new conda env.